src/styles/1-helpers/_variables.scss

```diff
@@ -0,0 +1,9 @@
+// colors
+$white: #fff;
+$blue: #00acdc;
+$black: #060b35;
+$grey: #616070;
+
+// general
+$fz: 12px;
+$lh: 18px;
```

Comment:
The same applies to these variables. You should not give them unreadable names, because now without context it is not clear what is stored in these variables.